### PR TITLE
{data}[foss/2020a,intel/2020a] netcdf4-python v1.5.3 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)
requires: https://github.com/easybuilders/easybuild-easyblocks/pull/2106 (otherwise the build will fail with MPI errors)

### CODE REVIEW

File: easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.3-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+name = 'netcdf4-python'
+version = '1.5.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://unidata.github.io/netcdf4-python/'
+toolchain = {'name': 'foss', 'version': '2020a'}
+description = """Python/numpy interface to netCDF."""
+
+toolchainopts = {'usempi': True}
```
Comment (smoors) [inline easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.3-foss-2020a-Python-3.8.2.eb:11]: style nit: move toolchain down:
```suggestion
description = """Python/numpy interface to netCDF."""

toolchain = {'name': 'foss', 'version': '2020a'}
toolchainopts = {'usempi': True}
```

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/9a6fb806d438271fcd9e9b374ba2ce80 for a full test report.

File: easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.3-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (Darkless012) [reply easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.3-foss-2020a-Python-3.8.2.eb]: fixed. Thanks

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 3 out of 4 (2 easyconfigs in this PR)
node375.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/02176212b5a4334a3952126e3eda10d2 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in this PR)
node375.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/a15db6df2eac762d4b5a1ca6beba679b for a full test report.

**Comment by smoors** [comment]
Going in, thanks @Darkless012!

**Pull request merged**